web_dashboard_tile/models/tile_tile.py

```diff
@@ -166,7 +138,7 @@ def open_link(self):
         }
         if self.action_id:
             res.update(self.action_id.read(
-                ['view_type', 'view_mode', 'view_id', 'type'])[0])
+                ['view_type', 'view_mode', 'type'])[0])
             # FIXME: restore original Domain + Filter would be better
```

Comment:
Done. I've added a new line to the readme too, because I fear without that comment the issue wasn't clear.